lldb/test/API/tools/lldb-dap/evaluate/TestDAP_evaluate.py

```diff
@@ -101,9 +101,9 @@ def run_test_evaluate_expressions(
         if context == "repl":
             # In the repl context expressions may be interpreted as lldb
             # commands since no variables have the same name as the command.
-            self.assertEvaluate("var", r"\(lldb\) var\n.*")
+            self.assertEvaluate("list", r"\(lldb\) list\n.*")
```

Comment:
Is the comment here now outdated?

Reply:
Its still applicable, we're just using a different lldb variable name to shadow `list` instead of `var`.

Reply:
Oh gotcha. I forgot that `list` is also a command